bricks/dart_frog_prod_server/hooks/lib/src/uses_workspace_resolution.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:io';
+import 'package:mason/mason.dart';
+import 'package:path/path.dart' as path;
+import 'package:pubspec_parse/pubspec_parse.dart';
```

Comment:
question:

Last time I check this package was unmaintained and had some open issues.

Probably out of the scope of this MR, but we might want to use an alternative for it.